software/acpid.txt

```diff
@@ -0,0 +1,123 @@
+well for is closing the lid. For some users this is enough. To run
+commands when the lid is closed, create an executable script at
+/etc/acpi/LID/00000080. The example below will suspend your computer to RAM.
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   #!/bin/sh                                                                  |
+|                                                                              |
+|   printf mem > /sys/power/state                                              |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+The rest of this article uses the more robust acpid program from the acpid package in the community repository. Install it as usual:
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   kiss build acpid                                                           |
+|   kiss install acpid                                                         |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+
+Identifying ACPI Events
+________________________________________________________________________________
+
+
+Enable acpid:
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ ln -s /etc/sv/acpid/ /var/service/                                       |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+
+Run acpi_listen to find events as they occur. For example, pressing the volume
+mute button will print something like this:
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ button/mute MUTE 00000080 00000000 K                                     |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+
+NOTE: Some 'Fn + *' keys might not appear as they are hardware-bound (e.g. WLAN
+      toggle). Just test them all and see what works. For most laptops, this
+      will also detect lid events (e.g. opening or closing).
+
+
+Read all events
+________________________________________________________________________________
+
+Create a file under /etc/acpi/events/ (just create these directories if they
+are missing) containing the following:
```

Comment:
```
Save the following lines to a file in /etc/acpi/events/. 
```

Is there a file name requirement also?

Reply:
I don't think so. The Arch wiki page doesn't mention anything about it. The event= line seems to determine everything